mobile/src/main/java/com/alexstyl/specialdates/util/Notifier.java

```diff
@@ -89,19 +93,53 @@ public void forDailyReminder(ContactEvents events) {
                 );
 
         String title = NaturalLanguageUtils.joinContacts(context, events.getContacts(), 3);
-        String fullText = TextUtils.join(", ", events.getContacts());
 
         NotificationCompat.Builder builder = new NotificationCompat.Builder(context)
                 .setSmallIcon(R.drawable.ic_stat_contact_event)
                 .setContentTitle(title)
                 .setLargeIcon(largeIcon)
-                .setStyle(new NotificationCompat.BigTextStyle().bigText(fullText))
                 .setVisibility(NotificationCompat.VISIBILITY_PRIVATE)
                 .setAutoCancel(true)
-                .setContentText(fullText)
                 .setContentIntent(intent)
+                .setNumber(events.getContactCount())
                 .setColor(context.getResources().getColor(R.color.main_red));
 
+        // if event.getContacts() > 1 then use InboxStyle, otherwise the BigTextStyle like here:
+        if (events.getContacts().size() == 1) {
+            int age = events.getContacts().get(0).getBirthday().getAgeOnYear(events.getDate().getYear());
+            String msg = context.getString(R.string.turns_age, age);
+
+            NotificationCompat.BigTextStyle bigTextStyle = new NotificationCompat.BigTextStyle().bigText(msg);
+            bigTextStyle.setBigContentTitle(title);
+            builder.setContentText(msg);
+
+            builder.setStyle(bigTextStyle);
+
+        } else if (events.getContacts().size() > 1) {
+            final NotificationCompat.InboxStyle inboxStyle = new NotificationCompat.InboxStyle();
```

Comment:
However, (since I added this to my Inspections) I see that a few lines lower (Notifier#getCircleBitmap()) several `final` keywords are used. What's your policy about `final`? Keep them? Remove them? Doesn't matter? :smiley: